targets/x86-64

```diff
@@ -2,6 +2,9 @@ config 'CONFIG_VDI_IMAGES=y'
 config 'CONFIG_VMDK_IMAGES=y'
 packages 'kmod-3c59x' 'kmod-8139cp' 'kmod-8139too' 'kmod-e100' 'kmod-e1000' 'kmod-e1000e' 'kmod-forcedeth' 'kmod-igb' 'kmod-natsemi' 'kmod-ne2k-pci'
 packages 'kmod-pcnet32' 'kmod-r8169' 'kmod-sis900' 'kmod-sky2' 'kmod-tg3' 'kmod-tulip' 'kmod-via-rhine' 'kmod-via-velocity'
+packages 'kmod-ath9k' 'kmod-ath10k'
```

Comment:
We should distinguish IBSS and 11s like in other targets and install kmod-ath10k-ct for IBSS. This is also missing the necessary ath10k firmware packages.

Reply:
i'll look into this.

Reply:
```
# opkg list | grep "ath10k-firmware" | awk '{ print $1 }'
ath10k-firmware-qca4019
ath10k-firmware-qca4019-ct
ath10k-firmware-qca4019-ct-htt
ath10k-firmware-qca6174
ath10k-firmware-qca9887
ath10k-firmware-qca9887-ct
ath10k-firmware-qca9887-ct-htt
ath10k-firmware-qca9888
ath10k-firmware-qca9888-ct
ath10k-firmware-qca9888-ct-htt
ath10k-firmware-qca988x
ath10k-firmware-qca988x-ct
ath10k-firmware-qca988x-ct-htt
ath10k-firmware-qca9984
ath10k-firmware-qca9984-ct
ath10k-firmware-qca9984-ct-htt
ath10k-firmware-qca99x0
ath10k-firmware-qca99x0-ct
ath10k-firmware-qca99x0-ct-htt
```

Reply:
question is, does it make sense to include all of those? at least i know of chips that work with  ath10k-firmware-qca9888 and ath10k-firmware-qca988x but better with the latter

Reply:
https://de.varia-store.com/produkt/31905-compex-wle600vx-802-11ac-a-b-g-n-minipci-express-karte.html

I own the WLE600VX and it identifies as `Network controller: Qualcomm Atheros QCA986x/988x 802.11ac Wireless Network Adapter`, seems to be a `QCA9882`. Is often sold with PC Engines APU2 I guess.